components/Graphs/shared/graph-resizer.tsx

```diff
@@ -0,0 +1,23 @@
+import { FiMaximize2 } from "react-icons/fi";
+
+export type OnToggleResizeEventType = (checked: boolean) => void;
+
+export const Resizer = ({ onToggleResize }: { onToggleResize: OnToggleResizeEventType }) => {
+  return (
+    <button
```

Comment:
This can probably use the Radix switch component in the future when I implement this for other graphs. It's still accessible, so all good.